sdk/tests/tutorials/properties/test_multivalue_properties.py

```diff
@@ -0,0 +1,110 @@
+# import general python packages
+import unittest
+import json
+import logging
+import uuid
+import pytz
+from datetime import datetime
+
+# import lusid specific packages
+import lusid
+import lusid.models as models
+from utilities import InstrumentLoader
+from utilities import TestDataUtilities
+
+# setup logging configuration
+logging.basicConfig(level=logging.INFO)
+
+
+class MultiLabelPropertyTests(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        # create a configured API client
+        api_client = TestDataUtilities.api_client()
+        cls.property_definitions_api = lusid.PropertyDefinitionsApi(api_client)
+        cls.instruments_api = lusid.InstrumentsApi(api_client)
+
+        cls.portfolios_api = lusid.PortfoliosApi(api_client)
+        cls.transaction_portfolios_api = lusid.TransactionPortfoliosApi(api_client)
+
+        # load instruments from InstrumentLoader
+        instrument_loader = InstrumentLoader(cls.instruments_api)
+        cls.instrument_ids = instrument_loader.load_instruments()
+        cls.portfolios_api = lusid.PortfoliosApi(api_client)
+
+    def get_guid(self):
+        # creates random alphanumeric code
+        return str(uuid.uuid4())[:12]
+
+    def test_create_portfolio_with_mv_property(self):
+        # Details of property to be created
+        uuid = self.get_guid()
+        effective_date = datetime(year=2018, month=1, day=1, tzinfo=pytz.utc)
+        scope = "MultiValueProperties"
+        code = "CallSchedule"
+        portfolio_code = f"ud-{uuid}"
```

Comment:
we don't need to create a new portfolio each time, its ok to reuse the same one

Reply:
Removing uuid and changing to static portfolio _code. 
`portfolio_code = "Portfolio-MVP"`